lib/Dialect/FIRRTL/FIRRTLOps.cpp

```diff
@@ -1922,6 +1922,24 @@ Value MemOp::getPortNamed(StringAttr name) {
   return Value();
 }
 
+static Value trackValueThroughAnoymousWires(Value value) {
+  auto wire = value.getDefiningOp<WireOp>();
+  // If the value is not a wire, return the value.
+  if (!wire)
+    return value;
+
+  // If the wire has a symbol, return the wire.
```

Comment:
It's not clear that symbols should matter here.

Reply:
Thanks, I changed to use `getModuleScopeDriver` so it is fixed